build.gradle

```diff
@@ -11,6 +12,7 @@ plugins {
 	id 'de.fuerstenau.buildconfig' version '1.1.8'
 	id "com.dorongold.task-tree" version "1.5"
 	id "com.geoffgranum.gradle-conventional-changelog" version "0.3.1"
+	id "org.asciidoctor.convert" version "2.4.0"
```

Comment:
this generates quite the few warnings - tried move it to latest `org.asciidoctor.jvm.convert` but then the whole include setup can't find their files.

Reply:
it is working now.